cmd/serve.go

```diff
@@ -53,12 +54,20 @@ var serveCmd = &cobra.Command{
 			log.Fatal("failed to create prowlarr client", zap.Error(err))
 		}
 
+		defaultSchemas := cfg.Storage.Schemas
+		if _, err := os.Stat(cfg.Storage.FilePath); err != nil {
+			if errors.Is(err, os.ErrNotExist) {
+				log.Debug("database does not exist, defaulting table values", zap.Any("schemas", cfg.Storage.TableValueSchemas))
```

Comment:
I think we'll probably want more intense version of this at some point (not just no db). But a future problem

Reply:
yeah agreed